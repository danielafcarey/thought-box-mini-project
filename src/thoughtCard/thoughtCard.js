import React from 'react';

export const ThoughtCard = ({ title, body }) => {
  return (
    <div>
      <h3>{ title }</h3> 
      <p>{ body }</p>
    </div>
  )
}
