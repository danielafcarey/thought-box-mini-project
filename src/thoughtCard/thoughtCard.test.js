import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import { ThoughtCard } from './thoughtCard.js';

describe('ThoughtCard', () => {

  it('should match the snapshot', () => {
    const mockProps = { title: 'title', body: 'body' }
    const thoughtCard = renderer.create(
      <ThoughtCard title={ mockProps.title } 
                   body={ mockProps.body }
      />);

    expect(thoughtCard).toMatchSnapshot();
  });

});
