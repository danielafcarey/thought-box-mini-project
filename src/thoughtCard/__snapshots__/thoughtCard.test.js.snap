// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThoughtCard should match the snapshot 1`] = `
<div>
  <h3>
    title
  </h3>
  <p>
    body
  </p>
</div>
`;
